kie-api/src/main/java/org/kie/api/management/ReleaseIdInfo.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.api.management;
+
+import javax.management.openmbean.CompositeData;
+
+import org.kie.api.builder.ReleaseId;
+
+public class ReleaseIdInfo implements ReleaseId {
```

Comment:
How about I rename it for what really is, `GAVCoordinates` ?

Then from an API user point of view, could easily realize `GAVCoordinates` class implements the ReleaseId interface if that is needed; otherwise, is immediately noticeable `GAVCoordinates` class has a method for comparing GAV coordinates to another .. ReleaseId. 

Chosen this name as acronym GAV have a wikipedia entry (in disambiguation) so it should be pretty known what actually a "GAV coordinate" is. If later this class could be helpful in other (public kie api) package can be moved, as long as I can continue enhance it as might be needed - unlikely to ReleaseId interface definition, unfortunately.

Before I go on an rename it, kindly asking for a feedback for this new name being ok please?

(PS: never a problem having these "nit-picking" comments ;) sorry just didn't realize it was ambiguous originally)